compiler/rustc_ast/src/ast.rs

```diff
@@ -107,11 +107,11 @@ impl PartialEq<Symbol> for Path {
     }
 }
 
-impl<CTX> HashStable<CTX> for Path {
+impl<CTX: rustc_span::HashStableContext> HashStable<CTX> for Path {
```

Comment:
Where is this impl used?

Reply:
This is needed to be able to call `ident.hash_stable`

Reply:
I mean the impl for Path.

Reply:
It's used by `AttrItem` and other AST structs